### PR TITLE
Sscs 10606 grant refused lta

### PR DESCRIPTION
**Pull request by nickhill111**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java [hunk 1]
```diff
@@ -93,32 +79,41 @@ public static void addDocumentToBundle(FooterService footerService, SscsCaseData
...
+        if (isPostHearingsEnabled && nonNull(postHearing.getReviewType())) {
+            return getPostHearingReviewDocumentType(postHearing);
+        }
+
+        return DocumentType.DECISION_NOTICE;
+    }
+
+    private static DocumentType getPostHearingReviewDocumentType(PostHearing postHearing) {
         PostHearingReviewType postHearingReviewType = postHearing.getReviewType();
-        if (isPostHearingsEnabled && nonNull(postHearingReviewType)) {
```
Comment (amoslv1) [inline src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java:97]: Why have you removed the post hearing flag?

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java [hunk 2]
```diff
@@ -171,16 +171,13 @@ protected PreSubmitCallbackResponse<SscsCaseData> issueDocument(Callback<SscsCas
...
-            PostHearing postHearing = caseData.getPostHearing();
-            if (nonNull(postHearing.getSetAside().getAction())) {
-                CcdCallbackMap action = postHearing.getSetAside().getAction();
-                boolean isGrantOrRefuseSetAsideAction = action.toString().equals(SetAsideActions.GRANT.getCcdDefinition())
-                        || action.toString().equals(SetAsideActions.REFUSE.getCcdDefinition());
-                embeddedDocumentTypeLabel =  isGrantOrRefuseSetAsideAction ? "Set Aside Decision Notice" : embeddedDocumentTypeLabel;
+            PostHearingReviewType postHearingReviewType = caseData.getPostHearing().getReviewType();
+
+            if (nonNull(postHearingReviewType)) {
+                return postHearingReviewType.getDescriptionEn() + " Decision Notice";
```
Comment (amoslv1) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java:180]: What is the reason for deleting this block of code? Has it been made redundant by getDescriptionEn?

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandlerTest.java [hunk 3]
```diff
@@ -154,18 +154,15 @@ void testDocumentPayloadValues() {
         assertThat(payload.getIdamSurname()).isEqualTo("Barry Allen");
     }
 
-    @ParameterizedTest
-    @EnumSource(value = SetAsideActions.class, names = {"GRANT", "REFUSE"})
-    void givenSetAsideState_andPostHearingsIsEnabled_thenReturnSetAsideDecisionNotice(SetAsideActions setAsideActions) {
```
Comment (amoslv1) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandlerTest.java:159]: Why is it no longer required to test grant and refused?

**Review by amoslv1 — COMMENTED** [review]
A few things I don't understand, could use some explanation please

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java (hunk 2 above)
Comment (nickhill111) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java:180]: yeah so before it was only changing it for set aside, we now want to change it for every journey

File: src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java (hunk 1 above)
Comment (nickhill111) [reply src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java:97]: i havent, its just the diff, its stayed on line 82

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandlerTest.java (hunk 3 above)
Comment (nickhill111) [reply src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandlerTest.java:159]: as it doesn't check the action anymore, it just checks the review type

**Review by amoslv1 — DISMISSED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Pull request merged**